### DISCUSSION TITLE
save button hidden

### DISCUSSION THREAD

**Discussion by solitartist** [aclap-dev/video-downloadhelper#1097] [Help and Q&A]
### Your question

I can choose the video, select download and convert on the second page, then the save button on the third page is hidden. I can resize (zoom out) and sometimes see the top of the save box, but most of the time it.s hidden no matter what I do. I'm on firefox, have a license, and refreshed a couple of days ago, Aug.
 2

### Addon & CoApp version and details

version: 9.1.0.41
channel: beta
build date: 2024-08-02
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 7503bc7b4b6c4977
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "win7",
    "arch": "x86_64",
    "node": "10"
  },
  "home": "C:\\Users\\smith",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running almost the same setup you are except I have the latest CoApp & you do not.  I've never seen what you're describing.  I suspect getting the new CoApp will not affect the problem . . . but you never know unless you try it.  You should get up to the latest CoApp in any case.  After all, you're running the latest beta of the browser extension.  You should match that for the CoApp.

Just as a general guideline, when you are reporting issues dealing with the way things look on your screen, you really ought to post a screenshot or two to illustrate the problem.  That would make things a lot clearer than words alone.

**Comment by paulrouget** [comment]
Like WildWilly said, please post a screenshot.

Also, can you try to run DownloadHelper in the sidebar?
